lib/src/components/NxFilterInput/__tests__/NxFilterInput.test.tsx

```diff
@@ -5,160 +5,109 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import { mount, shallow } from 'enzyme';
-import { faFilter, faSearch } from '@fortawesome/free-solid-svg-icons';
+import { waitFor } from '@testing-library/react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import userEvent from '@testing-library/user-event';
 
 import NxFilterInput, { Props } from '../NxFilterInput';
-import { PrivateNxTextInput } from '../../NxTextInput/NxTextInput';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import NxButton from '../../NxButton/NxButton';
-import Close from '../../../icons/Close';
 
 describe('NxFilterInput', function() {
   const minimalProps = { value: '' },
```

Comment:
If you assign minimalProps as type Props here, you would be able to get rid of <Props> in quickRender and renderEl below. Just a thought, and probably personal preference!

Reply:
Thanks, it is cleaner, updated here c79b2d1ca6a07085d952e9a2ff1a27c46df4219d.